rc/filetype/javascript.kak

```diff
@@ -55,21 +55,112 @@ define-command -hidden javascript-indent-on-char %<
 
 define-command -hidden javascript-insert-on-new-line %<
     evaluate-commands -draft -itersel %<
-        # copy // comments prefix and following white spaces
-        try %{ execute-keys -draft k <a-x> s ^\h*\K/{2,}\h* <ret> y gh j P }
+    execute-keys <semicolon>
+    # NOT working, copies single line comment "//"
```

Comment:
I missed those commented blocks when I merged, this needs to be fixed

Reply:
sorry, I was unable to get this working and overlooked it -- took the time to troubleshoot and it's now fixed. Actually the commented blocks mentioned below are what made it work.